editor/src/components/editor/server.ts

```diff
@@ -325,6 +326,40 @@ export async function saveThumbnail(thumbnail: Buffer, projectId: string): Promi
   }
 }
 
+export async function getUserPermissions(
```

Comment:
this _must_ check that the BFF is not enabled, because this would otherwise explode on production